src/view/style.rs

```diff
@@ -0,0 +1,150 @@
+use piet_scene::Color;
+
+use crate::{
+    id::Id,
+    widget::{
+        compose_style::{background::BackgroundWidget, padding::PaddingWidget},
+        Pod,
+    },
+    View,
+};
+
+pub fn padding<V: View<T, A>, T, A>(width: f64, view: V) -> impl View<T, A>
+where
+    V::Element: 'static,
+{
+    PaddingView::new(width, view)
+}
+
+pub struct PaddingView<V> {
```

Comment:
Here's an implicit decision about naming convention, that I really like;

It would be grammatically more obvious to name this `PaddedView`.

But I think this way it's way nicer, and we should adopt this as a naming convention.

E.g. `BorderView` instead of `BorderedView` etc.

This way it's a little easier to navigate the stuff xilem has to offer, even if your english is good. But more than anything, we make the library a little more accessible to non native speakers, who struggle with the language barrier